README.md

```diff
@@ -346,14 +346,15 @@ For example, in `ghcr.io/intertwin-eu/itwinai:0.2.2-torch2.6-jammy`:
 The `TAG` follows the convention:
 
 ```text
-X.Y.Z-[torch|tf]x.y-distro
+[jlab-]X.Y.Z-(torch|tf)x.y-distro
```

Comment:
In this case, `(torch|tf)` is an "XOR", right? You can pick one or the other, but not both? Not super important but could consider mentioning it?